nosql/assignment.nosql.schema.txt

```diff
@@ -0,0 +1,84 @@
+
+Collection - Users
+
+{
+	account: {
+		username: "shirish.sp",
+		password: "**********",
+		roles: [
```

Comment:
Use Case: 
- Admin would have access only to a subset of users. And he would be able to manage only those users, and those posts of those users assigned to him 
- Similarly, an editor can edit only those posts which are assigned to him